### PR TITLE
Create goober.json

### PR DESCRIPTION
**Pull request by JasonESMIN**

### CODE REVIEW

File: goober.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "wisdom": 4,
+    "charisma": 15
+  },
+  "attack": {
+    "name": "Booger sneeze",
+    "damage": 500
+  },
+  "special": {
+      "name": "Diarrhea Combustion",
+      "description": "The Taco Bell fumes burn the enimies",
```
Comment (JackRichard889) [inline goober.json]: "enimies" should be spelled "enemies."

**Review by xombotron — COMMENTED** [review]
please change your special attack

**Comment by xombotron** [comment]
please change the effect

**Comment by mohammedarts** [comment]
> please change the effect
The creator says no

**Comment by xombotron** [comment]
aight

**Pull request closed without merging**